WordPress/Classes/ViewRelated/Reader/ReaderMenuViewModel.swift

```diff
@@ -403,7 +409,18 @@ enum ReaderDefaultMenuItemOrder: Int {
     /// - Returns: The requested menu item or nil.
     ///
     func menuItemForTagAtIndex(index: Int) -> ReaderMenuItem? {
-        guard let topic = tagsFetchedResultsController.objectAtIndexPath(NSIndexPath(forRow: index, inSection: 0)) as? ReaderAbstractTopic else {
+        var fetchedIndex = index
+        if ReaderHelpers.isLoggedIn() {
+            if fetchedIndex == 0 {
+                let title = NSLocalizedString("Add a Tag", comment: "Title. Let's the user know that they can use this fature to subscribe to new tags.")
```

Comment:
Localization desc, `Let's the user know that they can use this fature`, maybe `Let's` > `Let` and `fature`.